src/elm/4_input/4_input.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Input fra brukeren"
+level: 3
+language: nb-NO
+author: Alexander Perry
+---
+
+# Introduksjon {.intro}
+
+Input-felter lar brukere skrive inn tekst, tall, datoer og så videre i nettsiden vår. Informasjonen vi får fra brukerne kan vi endre, skrive ut, og kombinere med annen informasjon.
+
+# Hente tekst fra brukeren {.activity}
+
+## Moduler og program {.check}
+
+- Øverst i koden vår må vi si hvilke *moduler* vi skal bruke
+
+```elm
+import Html exposing (Html, Attribute, beginnerProgram, text, div, input, button)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onInput, onClick)
+import String
+```
+
+## Modell, meldinger og oppdatering {.check}
+
+- Vi oppretter programmet vårt med en *modell*, et *view* og en *update*-funksjon. Vi kommer tilbake til hvordan vi oppretter disse, skriv inn følgende i koden:
```

Comment:
Endret teksten til "Vi oppretter programmet vårt med en *modell* som inneholder data (`model`), et *brukergrensesnitt* hvor vi viser frem data (`view`) og en funksjon for *oppdatering* av data (`update`)."